docs/starters.yml

```
@@ -1,3 +1,22 @@
+- url: https://gatsby-wordpress-libre.netlify.com/
+  repo: https://github.com/armada-inc/gatsby-attila-theme-starter
+  description: A Gatsby theme plugin for creating blogs from headless WordPress CMS.
```

```suggestion
  description: A Gatsby starter for creating blogs from headless WordPress CMS.
```

If this is to be a starter, I think it would make sense to clarify this wording 🙂

---

Yeah it makes sense 🙂 I have made changes that you suggested. will do same for #22175